Frontend-v1-Original/components/ssSwap/ssSwap.tsx

```diff
@@ -6,7 +6,7 @@ function Swap() {
     <div className="mt-32 flex h-full min-h-[calc(100vh-432px)] w-full flex-col items-center justify-evenly sm:mt-0 lg:flex-row">
       <Paper
         elevation={0}
-        className="flex w-full max-w-[485px] flex-col bg-transparent p-3 shadow-lg shadow-cantoGreen lg:p-6"
+        className="flex w-full max-w-[485px] flex-col bg-transparent p-3 shadow-glow shadow-cantoGreen lg:p-6"
```

Comment:
you don't need shadow-cantoGreen, because shadow-glow already has it in tailwind config. We can rename shadow-glow to shadow-glowCantoGreen in this case so we know it's glowing with cantoGreen color.

```suggestion
        className="flex w-full max-w-[485px] flex-col bg-transparent p-3 shadow-glowCantoGreen lg:p-6"
```